### PR TITLE
Fixed Typographical Error in guide/english/blockchain/index.md

### PR DESCRIPTION
**Pull request by iamrajiv**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

File: guide/english/blockchain/index.md [hunk 1]
```diff
@@ -65,13 +65,13 @@ A third party is usually needed to validate trust relationships. In the most com
 For this reason, Blockchain is also known as **"The Trust Protocol"**.
 
 ## History
-The first work on a cryptographically secured chain of blocks was described in 1991 by Stuart Haber and W. Scott Stornetta. They wanted to implement a system where documents' timestamps could not be tampered with or backdated. In 1992, Bayer, Haber, and Stornetta incorporated Merkle trees to the design, which improved its efficiency by allowing several documents to be collated into one block.
+The first work on a cryptographically secured chain of blocks as described in 1991 by Stuart Haber and W. Scott Stornetta. They wanted to implement a system where documents' timestamps could not be tampered with or backdated. In 1992, Bayer, Haber, and Stornetta incorporated Merkle trees to the design, which improved its efficiency by allowing several documents to be collated into one block.
```
Comment (Manish-Giri) [inline guide/english/blockchain/index.md]: > The first work on a cryptographically secured chain of blocks **was** described in 1991 by Stuart Haber and W. Scott Stornetta.

In this sentence, "was" acts as the verb. "as" can't work as a verb here. Please revert to "was".

You can add a hyphen here if you wish - "cryptographically secured" .

File: guide/english/blockchain/index.md [hunk 2]
```diff
@@ -83,28 +83,28 @@ In May 2018, Gartner found that only 1% of CIOs indicated any kind of blockchain
 
 ## Types of Blockchains
 Blockchains can be broadly classified into two categories -
-1. Permissionless, public blockchains - these are not controlled by any single company or person. The blockchain is accessible to anyone to participate in. Bitcoin and Ethereum are examples of permissionless blockchains. Consensus is arrived at by participating miner nodes in the network, who perform certain actions (Proof-of-Work) and are rewarded for helping to maintain the state of the distributed ledger. Anyone interested can become a miner and participate in the consensus building process. 
+1. Permissionless, public blockchains - these are not controlled by any single company or person. The blockchain is accessible to anyone to participate in. Bitcoin and Ethereum are examples of permissionless blockchains. The consensus is arrived at by participating miner nodes in the network, who perform certain actions (Proof-of-Work) and are rewarded for helping to maintain the state of the distributed ledger. Anyone interested can become a miner and participate in the consensus building process.
```
Comment (Manish-Giri) [inline guide/english/blockchain/index.md]: "The" isn't required before consensus. Please revert.

File: guide/english/blockchain/index.md [hunk 3]
```diff
@@ -65,13 +65,13 @@ A third party is usually needed to validate trust relationships. In the most com
...
+The words block and chain were used separately in Satoshi Nakamoto's original paper but were eventually popularized as a single word, the blockchain, by 2016. The term blockchain 2.0 refers to new applications of the distributed blockchain database, first emerging in 2014. The Economist described one implementation of this second-generation programmable blockchain as coming with "a programming language that allows users to write more sophisticated smart contracts, thus creating invoices that pay themselves when a shipment arrives or share certificates which automatically send their owners dividends if profits reach a certain level."
```
Comment (Manish-Giri) [inline guide/english/blockchain/index.md]: > as a single word, **the** blockchain, by 2016.

Again, "the" isn't required here. You can capitalize blockchain if you wish.

File: guide/english/blockchain/index.md [hunk 4]
```diff
@@ -83,28 +83,28 @@ In May 2018, Gartner found that only 1% of CIOs indicated any kind of blockchain
...
 
-Another issue is its lack of regulation, or a central authority backing its currencies, which in the past has lead to huge increases in speculation, and ultimately crashes. An example of this, is the huge surge in the value of bitcoin, the poster child of blockchain, which after surging to a new high in January 2018, fell by around 75% by August of the same year. This left many eager speculators ruined. However, this is not an inherent problem with the technology itself, being more a reflection of peoples desires to get rich quick than the evil of blockchain.
+Another issue is its lack of regulation, or a central authority backing its currencies, which in the past has lead to huge increases in speculation, and ultimately crashes. An example of this is the huge surge in the value of bitcoin, the poster child of the blockchain, which after surging to a new high in January 2018, fell by around 75% by August of the same year. This left many eager speculators ruined. However, this is not an inherent problem with the technology itself, being more a reflection of peoples desires to get rich quick than the evil of blockchain.
```
Comment (Manish-Giri) [inline guide/english/blockchain/index.md]: > poster child of **the** blockchain

Remove "the".

**Review by Manish-Giri — CHANGES_REQUESTED** [review]

**Comment by iamrajiv** [comment]
@Manish-Giri is this PR good to be merged

**Review by Manish-Giri — APPROVED** [review]
LGTM. Thank You for your contribution.

**Pull request merged**